common/account/faq.mdx

```diff
@@ -13,5 +13,8 @@ You can delete your credit card from `Account` > `Billing` page. However, you sh
 ## How can I change my email address?
 We are working on a solution which will allow you to change your email address on console. Until then, please contact us at [support@upstash.com](mailto:support@upstash.com) to change your email address.
 
+## Can I set an upper spending limit, so I don't get surprises after an unexpected amount of high traffic?
+On Pay as You Go model, you can set a budget for your Redis and Kafka instances. When your monthly cost reaches the max budget, we send an email to inform you and throttle your database. You will not be charged beyond your set budget.
```

Comment:
In the header we say database and kafka but in the explanation we only mention database.
Fact: we do throttle databases but we don't throttle kafka clusters automatically.
Another fact: in the mails sent we say that we will throttle both.
Just wanted to point out 😄